search-api.md

```diff
@@ -88,6 +90,15 @@ After receiving a request, the remote server can respond in one of two ways:
 }
 ```
 
+#### Disclaimer
+* *Optional*
+* Disclaimer text for the service, this is optional and can be included in all requests and responses. If included it is assumed that it supersedes the disclaimer stored in the GitHub repository.
+* Note that the diclaimer should be shown with the returned data per item 6 of the MME service agreement.
```

Comment:
This is very good point, we have a EULA and guidelines in the FAQ they have to agree to when they sign up. I will check with Ada if we need to add disclaimers for each MME.